freesound/local_settings.example.py

```diff
@@ -1,43 +1,22 @@
 # -*- coding: utf-8 -*-
 
-
-# -------------------------------------------------------------------------------
-# Required Django settings
-
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': '',                      # Or path to database file if using sqlite3.
-        'USER': '',                      # Not used with sqlite3.
-        'PASSWORD': '',                  # Not used with sqlite3.
-        'HOST': '',                      # Set to empty string for localhost. Not used with sqlite3.
-        'PORT': '',                      # Set to empty string for default. Not used with sqlite3.
-    }
-}
-
-
-# -------------------------------------------------------------------------------
-# Some default configurations for development (no need to make changes here)
+import os
 
 DEBUG = True
 DISPLAY_DEBUG_TOOLBAR = True
 
-SECRET_KEY = 'thisisasecretkey'
-
-# Default SOLR URLs if running locally
-SOLR_URL = "http://localhost:8983/solr/fs2/"
-SOLR_FORUM_URL = "http://localhost:8983/solr/forum/"
 
-# Default Gearman hostname:port if running locally
-GEARMAN_JOB_SERVERS = ["localhost:4730"]
+# Use email file backend
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
+EMAIL_FILE_PATH = os.path.join(os.path.abspath(os.path.join(os.path.dirname(__file__), '../freesound-data/')), "_mail/")
 
-# Set data URL to production Freesound as data files will not likely be in local
-DATA_URL = "https://freesound.org/data/"
+# Use cache file backend
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache', # 'django.core.cache.backends.dummy.DummyCache',  # 'django.core.cache.backends.locmem.LocMemCache',  #
```

Comment:
out of interest, any specific why you're using the file cache here? During dev I'd suggest that we use either the dummy cache (so that we can identify cases where we're not doing sql queries properly), or django's in-memory development cache

Reply:
I'm not sure, I keep on changing but I can't remember why. Probably because I can manage cache with file manager. I'll change it to locmem by default.